addons/mrp/report/mrp_production_templates.xml

```diff
@@ -41,18 +45,24 @@
                             <span t-field="o.product_qty"/>
                             <span t-field="o.product_uom_id.name" groups="uom.group_uom"/>
                         </div>
+                         <div class="col-3" t-if="o.qty_producing">
+                            <strong>Quantity Producing:</strong><br/>
+                            <span t-field="o.qty_producing"/> Units
+                        </div>
                     </div>
 
                     <div t-if="o.workorder_ids" groups="mrp.group_mrp_routings">
                         <h3>
                             <span t-if="o.state == 'done'">Operations Done</span>
-                            <span t-else="">Operations Planned</span>
+                            <span t-elif="o.is_planned">Operations Planned</span>
+                            <span t-else="">Operations</span>
                         </h3>
                         <table class="table table-sm">
                             <tr>
                                 <th><strong>Operation</strong></th>
                                 <th><strong>WorkCenter</strong></th>
-                                <th><strong>No. Of Minutes</strong></th>
+                                <th><strong>Duration(minutes)</strong></th>
+                                <th t-if="o.state in ('progress', 'to_close')"><strong>Actual Duration(minutes)</strong></th>
```

Comment:
`Duration(minutes)` => missing a space between Duration and (minutes) in both cases